docs/reference/platforms/statsig.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Statsig
+nav_order: 7
+parent: Platforms which perform SRM checks natively
+grand_parent: Reference
+---
+
+# Statsig
+
+Default SRM Checks
+{: .label .label-green }
+
+
+[Statsig](https://www.statsig.com) automatically and continuously verifies and warns users about Sample Ratio Mismatch. Documentation is [available here](https://docs.statsig.com/experiments-plus/monitor).
+
+![A screenshot of Statsig's diagnostics interface](https://user-images.githubusercontent.com/74588208/198129020-3620bea8-fabb-42c1-b30c-6f5c61ce370f.png)
```

Comment:
@vijaye-statsig Why did you not include the image in the commit instead of referencing an external resource?

This is fine for now, but I might replace it with a local copy of the file later.